ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationSidebar.tsx

```diff
@@ -95,10 +107,17 @@ function NavigationSidebar({
                     path={violationsBasePath}
                     title={basePathToLabelMap[violationsBasePath]}
                 />
+                {isComplianceEnhancementsEnabled && (
+                    <LeftNavItem
```

Comment:
Not for this PR, can you follow up with Doron and Kylie whether this needs `<BadgedNavItem variant="TechPreview"` like Workload CVEs below?